server/serve.py

```diff
@@ -1,29 +1,49 @@
-from flask import Flask, request, render_template, jsonify
+import time
+from flask import Flask, request,  jsonify
 
 app = Flask(__name__, template_folder='template')
-gpio_state = {'gpioState': []}
 
-@app.route('/gpio_state', methods=['POST'])
+state={}
+
+@app.route('/hooks', methods=['POST'])
 def receive_data():
+    
+    lastUpdate = time.time()
+    global state
     try:
-        content = request.get_json()
-        received_gpio_state = content['gpioState']
-        gpio_state['gpioState'] = received_gpio_state
+
+        content = request.get_json() 
+        isinstance(content.get('state'), dict)

```

Comment:
Po co jest ta linijka?

Reply:
> Po co jest ta linijka?

no właściwie to po nic , równie dobrze mogłoby jej nie być